manifests/prometheus-adapter-deployment.yaml

```diff
@@ -7,15 +7,20 @@ spec:
   replicas: 1
   selector:
     matchLabels:
-      name: prometheus-adapter
+      app.kubernetes.io/component: exporter
```

Comment:
Is this exporter category?

Reply:
good catch, it should be sth else. Any ideas?

Reply:
Monitoring Adapter might be good? @s-urbaniak @dgrisonnet any more ideas?

Reply:
If we want to be general, we can just say it's an `adapter` but if we want to me more precise, I think `metrics-adapter` would be appropriate since its an adapter implementing the resource, custom, and external metrics APIs.

Reply:
I looked in other metrics API servers and did not find good precedences either. k8s-cloudwatch-adapter uses `adapter` (https://github.com/awslabs/k8s-cloudwatch-adapter/blob/765b3392fad9d120d1a29c592244108e1c73645c/charts/k8s-cloudwatch-adapter/templates/_helpers.tpl#L53), all the other implementations don't provide that label.

@dgrisonnet we can potentially also suggest `metrics-adapter` for metrics-server in https://github.com/kubernetes-sigs/metrics-server/blob/master/manifests/base/deployment.yaml